modules/openapi-generator/src/main/resources/kotlin-client/model_room_init_var.mustache

```diff
@@ -1 +1 @@
-{{#isNullable}}?{{/isNullable}}{{^required}}?{{/required}}{{#defaultvalue}} = {{defaultvalue}}{{/defaultvalue}}{{^defaultvalue}}{{#isNullable}} = null{{/isNullable}}{{^required}} = null{{/required}}{{/defaultvalue}}
\ No newline at end of file
+{{#isNullable}}?{{/isNullable}}{{^required}}{{^isNullable}}?{{/isNullable}}{{/required}}{{#defaultvalue}} = {{defaultvalue}}{{/defaultvalue}}{{^defaultvalue}}{{#isNullable}} = null{{/isNullable}}{{^required}}{{^isNullable}} = null{{/isNullable}}{{/required}}{{/defaultvalue}}
```

Comment:
@mactho could you please change the order of the conditions in both changes to be easier to read please?
`{{^isNullable}}{{^required}}?{{/required}}{{/isNullable}}`
This way it becomes,
```
if isNullable {
} else {
    if !required {
    }
}
```

Reply:
I'll update it after merging this PR into master.